fhir/src/main/java/gov/cms/ab2d/fhir/Versions.java

```diff
@@ -0,0 +1,304 @@
+package gov.cms.ab2d.fhir;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.context.FhirVersionEnum;
+import lombok.extern.slf4j.Slf4j;
+
+import java.lang.reflect.Method;
+import java.util.*;
+
+/**
+ * This class provides a lot of underlying methods for the other Util classes including:
+ *   1. Defining supported versions
+ *   2. Defining valid classes
+ *   3. Creating enumerations of versions and mapping between FHIR context objects and those versions
+ *   4. Package private methods to use reflection to instantiate classes, enums and to invoke set/get for those classes
+ *   5. The ability to determine which version is being used based on the URL passed from the HttpRequest
+ */
+@Slf4j
+public class Versions {
+    public enum FhirVersions {
+        R3,
```

Comment:
Should this be STU3? R3 isn't really a version.

Reply:
ok